pkg/registry/apis/provisioning/jobs/pullrequest/render.go

```diff
@@ -36,14 +37,22 @@ func (r *screenshotRenderer) IsAvailable(ctx context.Context) bool {
 	return r.render != nil && r.render.IsAvailable(ctx) && r.blobstore != nil && r.isPublic
 }
 
-func (r *screenshotRenderer) RenderDashboardPreview(ctx context.Context, namespace, repoName, path, ref string) (string, error) {
-	url := fmt.Sprintf("admin/provisioning/%s/dashboard/preview/%s?kiosk&ref=%s", repoName, path, ref)
```

Comment:
can you add unit tests for this one. Seems easy to get it wrong?

Reply:
I will add unit tests in the next PR -- that updates this to work without the image renderer configured.